pkg/csi_driver/utils.go

```diff
@@ -495,7 +509,7 @@ func isSidecarVersionSupportedForGivenFeature(imageName string, sidecarMinSuppor
 	isManagedSidecar := re.MatchString(imageName)
 
 	if !isManagedSidecar {
-		klog.Infof("mountOptions should not be passed because this is a private sidecar image %q", imageName)
+		klog.Infof("mountOptions should not be passed unless identity provider is specified by user, because this is a private sidecar image %q", imageName)
```

Comment:
this function isSidecarVersionSupportedForGivenFeature  is also used for other feature checks https://github.com/GoogleCloudPlatform/gcs-fuse-csi-driver/blob/main/pkg/csi_driver/node.go#L357

We can make the klog message generic, "{sidecarMinSupportedVersion} check skipped since this {imageName} is not a GKE managed image"